webapp/app/model/form_data.py

```diff
@@ -157,6 +157,18 @@ def account_holder_must_be_declared(cls, v, values):
                 raise MissingError
         return v
 
+    @validator('person_a_requests_pauschbetrag', always=True)
+    def person_a_required_has_disability(cls, v, values):
+        if not values.get('person_a_has_disability') == 'yes' and not v:
+            raise MissingError
+        return v
+    
+    @validator('person_b_requests_pauschbetrag', always=True)
+    def person_b_required_has_disability(cls, v, values):
+        if show_person_b(values.get('familienstandStruct', {})) and not values.get('person_b_has_disability') == 'yes' and not v:
+            raise MissingError
+        return v
```

Comment:
I think we can omit the first check as this should already be covered by the check for `person_b_has_disability`? So this should be enough:
```
if values.get('person_b_has_disability') == 'yes' and not v:
    raise MissingError
return v
```

What do you think?

Reply:
Thanks :)